x-pack/plugins/translations/translations/fr-FR.json

```
@@ -43802,8 +43801,8 @@
     "xpack.slo.sloEmbeddable.config.sloSelector.placeholder": "Sélectionner un SLO",
     "xpack.slo.sloEmbeddable.displayName": "Aperçu du SLO",
     "xpack.slo.sloEmbeddable.overview.sloNotFoundText": "Le SLO a été supprimé. Vous pouvez supprimer sans risque le widget du tableau de bord.",
-    "xpack.slo.sLOGridItem.targetFlexItemLabel": "Cible {target}",
     "xpack.slo.sloGridItem.targetFlexItemLabel": "Cible {target}",
+    "xpack.slo.sLOGridItem.targetFlexItemLabel": "Cible {target}",
```

Q: why is this change needed? Can we keep it for consistency with the other fields
```suggestion
    "xpack.slo.sloGridItem.targetFlexItemLabel": "Cible {target}",
```

---

I don't know... just ran the script to fix `i18n` and it automatically applied these changes. I won't change it because of that.